dev/tests/functional/lib/Magento/Mtf/Util/Generate/Fixture/FieldsProvider.php

```diff
@@ -101,13 +95,7 @@ protected function flatCollectFields(array $fixture)
 
         $attributes = [];
         foreach ($fields as $code => $field) {
-            $attributes[$code] = array(
-                'attribute_code' => $code,
-                'backend_type' => $field['DATA_TYPE'],
-                'is_required' => ($field['PRIMARY'] || $field['IDENTITY']),
-                'default_value' => $field['DEFAULT'],
-                'input' => ''
-            );
+            $attributes[$code] = ['attribute_code' => $code, 'backend_type' => $field['DATA_TYPE'], 'is_required' => ($field['PRIMARY'] || $field['IDENTITY']), 'default_value' => $field['DEFAULT'], 'input' => ''];
```

Comment:
I'm totally against this.

```php
            $attributes[$code] = array(
                'attribute_code' => $code,
                'backend_type' => $field['DATA_TYPE'],
                'is_required' => ($field['PRIMARY'] || $field['IDENTITY']),
                'default_value' => $field['DEFAULT'],
                'input' => ''
            );
```
```php
            $attributes[$code] = ['attribute_code' => $code, 'backend_type' => $field['DATA_TYPE'], 'is_required' => ($field['PRIMARY'] || $field['IDENTITY']), 'default_value' => $field['DEFAULT'], 'input' => ''];
```

This version is better no?
```php
            $attributes[$code] = [
                'attribute_code' => $code,
                'backend_type'   => $field['DATA_TYPE'],
                'is_required'    => ($field['PRIMARY'] || $field['IDENTITY']),
                'default_value'  => $field['DEFAULT'],
                'input'          => '',
            ];
```

Reply:
I know. Should be fixed with php-cs-fixer/phpcbf.

Reply:
Excluded dev for now.